### ISSUE TITLE
DLL threads are not kept alive by the debugger

### ISSUE THREAD

**Issue by faltur** [x64dbg/x64dbg#366]
When I open this simple DLL (http://pastebin.com/kzuPyfVL) in the x32dbg it abruptly terminates the debugging without keeping the thread alive. The disasm, stack, registers and Threads windows are all cleared, not giving me the opportunity to see the latest data from them. The status bar says: "Terminated" - "Debugging stopped!". From the log window:
....
Thread 15FC exit
Thread 1B24 created
Process stopped with exit code 0x1
Saving database...16ms
Debugging stopped!

Tested using the latest snapshot (snapshot_2015-08-14_02-23), Windows 7 SP1 32 bits.

**Comment by mrexodia** [comment]
I don't believe this is a bug. x64dbg just follows the path of execution and if the DLL is not kept alive by something it will terminate.

**Comment by Nukem9** [comment]
The thread is still running and alive at termination.
https://bitbucket.org/mrexodia/titanengine-update/src/51208e22f3354be8aa4f9926b22cb683b174e431/TitanEngineLoaders/LibraryLoader/LibraryLoader.cpp?at=master#LibraryLoader.cpp-22

DllMain returns -> main() returns -> ExitProcess called, `TestProcThread` is still running (forcefully terminated)
